contracts/EpochTokenLocker.sol

```diff
@@ -31,9 +31,15 @@ contract EpochTokenLocker {
         uint amount
     );
 
+    struct AccountInformation {
+        bool isAddedToGlobalAccountsList;
+        mapping(address => BalanceState) balances;
```

Comment:
Is this approach better than keeping an id for every user (right now is like that but implicit --> position in the array), then keeping a map of the id of every user `mapping(address => uint)`

You know how many users you have, because you have the size of the array.

Reply:
I think your suggestion would be slightly more efficient since it would only require one extra word (the ID) to be stored, whereas here we require two words (isAddedToGlobalAccountsList, address in vector).

I'm working on using a Set library instead of a hand crafted solution in the smart contract, which should make reading this easier I hope.

Reply:
It was fine to read, hehe
anyways, tested libs are always good :)